README.md

```diff
@@ -44,6 +44,24 @@ The import process takes around 10 minutes to complete.
 node server.js // Default environment is development
 ```
 
+### Running with Docker
+
+Postcodes.io is packaged as a Docker container identified on the Docker Hub as `james/postcodes.io`.
```

Comment:
This would probably need to be changed once merged right?

Reply:
@billinghamj this is addressed in the notes above.

> Note: The automated Docker Hub build james/postcodes.io is currently linked to jamescun/postcodes.io. If merged, either Docker Hub james/postcodes.io will need to be altered to ideal-postcodes/postcodes.io or a new Docker Hub repository needs to be created and the README updated.

Reply:
👍🏼